akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -68,6 +72,11 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
     "contain the class name where the first log was called" in {
       val eventFilter = EventFilter.custom({
         case l: LogEvent if l.logClass == classOf[ActorLoggingSpec] ⇒ true
+        case l: LogEvent if isScala211 ⇒
+          // TODO remove in Akka 2.6 when we drop Scala 2.11
+          // the class with 2.11 is like
+          // ActorLoggingSpec$$anonfun$1$$anonfun$apply$mcV$sp$26$$anonfun$apply$6$$anonfun$apply$7
+          l.logClass.getName.startsWith(classOf[ActorLoggingSpec].getName)
         case l: LogEvent ⇒
           println(l.logClass)
```

Comment:
'println' in a test? I realize it is out of scope but that's odd?

Reply:
likely there as the test is using a TestEventListener

Reply:
and it would only print if the test fails, and then it's useful